styles/theme.ts

```diff
@@ -26,9 +26,9 @@ const theme = createTheme({
 		},
 		secondary: {
 			main: "#B3BDFF",
-			swap: "#B3BDFF",
-			pool: "#E4D1FF",
-			bridge: "#B1EAEB",
+			swap: "rgba(17, 48, 255, 0.35)",
+			pool: "rgba(152, 71, 255, 0.35)",
+			bridge: "rgba(45, 200, 203, 0.35)",
 		},
```

Comment:
are they quite different between the HEX and RGBA values?

Reply:
same color just allows for the alpha channel/transparency when used as a backdrop